cmd/kubeadm/app/apis/kubeadm/install/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package install // import "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/install"
```

Comment:
https://github.com/kubernetes/kubernetes/issues/29014 and https://github.com/kubernetes/kubernetes/pull/29042

Seems like it's half-finished though, since no new doc.go files were added

Ok with leaving as-is?